altapay.php

```diff
@@ -2535,4 +2505,23 @@ public function returnError($paymentID, $exception)
 
         return $cookie->altapayError;
     }
+
+    /**
+     * @param int $itemID
+     * @param int|float $cmsTotal
+     * @param int|float $gatewayTotal
```

Comment:
I don't think there is a point in hinting both int and float in PHP?
```suggestion
     * @param float $cmsTotal
     * @param float $gatewayTotal
```